content/docs/guides/converting-gatsby-to-tina.mdx

```diff
@@ -39,8 +38,8 @@ npx @tinacms/cli@latest init
 After running the command above you'll receive a few prompts
 
 1. When prompted to select a framework select `other`
-2. Choose `NPM` as your package manager
-3. When asked if you'd like to use Typescript choose `no`
+2. Choose `yarn` as your package manager
```

Comment:
Could we instead of yarn just put a generic choice option here? 

```suggestion
2. Choose your preferred package manager for this project.
```

Reply:
I'm not sure if it'll work with other package managers because it uses the resolutions array. 

https://stackoverflow.com/questions/52416312/npm-equivalent-of-yarn-resolutions